File: client-app/src/app/models/bug.ts
import { Interface } from "readline";

export interface IBug {
  id: string;
  title: string;
  description: string;
  createdOn: string;
  admin: string;
  developer: string;
  category: string;
  priority: string;
  status: string;
  resolvedOn: string;
}

export default IBug;

File: Domain/Bug.cs
using System;

namespace Domain
{
    public class Bug
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime CreatedOn { get; set; }
        public string Admin { get; set; }
        public string Developer { get; set; }
        public string Category { get; set; }
        public string Priority { get; set; }
        public string Status { get; set; }
        public DateTime? ResolvedOn { get; set; }
    }
}